arduino/sensors/Kconfig

```
@@ -4,6 +4,7 @@ source "$PKGS_DIR/packages/arduino/sensors/Adafruit-Sensor/Kconfig"
 source "$PKGS_DIR/packages/arduino/sensors/Adafruit-AHTX0/Kconfig"
 source "$PKGS_DIR/packages/arduino/sensors/Adafruit-DHT/Kconfig"
 source "$PKGS_DIR/packages/arduino/sensors/Adafruit-DPS310/Kconfig"
+source "$PKGS_DIR/packages/arduino/sensors/Adafruit-SHT31-D/Kconfig"
```

这个名字有问题，你的文件夹叫Adafruit-SHT31结果这里的路径写成了Adafruit-SHT31-D，严格按照文件夹的名字来。改为：
```suggestion
source "$PKGS_DIR/packages/arduino/sensors/Adafruit-SHT31/Kconfig"
```